container_pipeline/lib/default_settings.py

```diff
@@ -0,0 +1,68 @@
+LOG_LEVEL = 'DEBUG'
+
+LOG_PATH = '/srv/pipeline-logs/cccp.log'
+
+LOGGING_CONF = dict(
+    version=1,
+    level=LOG_LEVEL,
+    formatters=dict(
+        bare={
+            "format": ('[%(asctime)s] %(name)s p%(process)s %(lineno)d '
+                       '%(levelname)s - %(message)s')
+        },
+    ),
+    handlers=dict(
+        console={
+            "class": "logging.StreamHandler",
+            "formatter": "bare",
+            "level": LOG_LEVEL,
+        },
+        log_to_file={
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': LOG_PATH,
+            'mode': 'a+',
+            'formatter': 'bare',
+        }
+    ),
+    loggers={
+        'linter': {
```

Comment:
Should we use `dockerfile-linter` here instead? Support for other image formats might come up in future, right? cc: @bamachrn